react-todo-app/src/pages/About.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Link, useRouteMatch, Route } from "react-router-dom"
```

Comment:
- Same thing here, In react-router-dom v6, replace `useRouteMatch` with `useMatch`.
for more info, you can check this [Link](https://reactrouter.com/docs/en/v6/upgrading/v5#replace-useroutematch-with-usematch)